goalc/main.cpp

```diff
@@ -45,15 +46,24 @@ int main(int argc, char** argv) {
   app.add_flag("--user-auto", auto_find_user,
                "Attempt to automatically deduce the user, overrides '-user'");
   app.add_option("-g,--game", game, "The game name: 'jak1' or 'jak2'");
+  app.add_option("--proj-path", project_path_override,
+                 "Specify the location of the 'data/' folder");
   app.validate_positionals();
   CLI11_PARSE(app, argc, argv);
 
-  if (!file_util::setup_project_path(std::nullopt)) {
-    return 1;
-  }
```

Comment:
you can't get rid of this completely, this needs to still run if a project path is not provided, change it to an `else if` and add it after the condition you added below.  With that change, seems to work as expected
![image](https://user-images.githubusercontent.com/13153231/191116577-1773f89f-cbee-41a9-b1e8-f353ad56af88.png)